src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/controller/OrchestrationSessionsController.kt

```diff
@@ -110,7 +110,7 @@ class OrchestrationSessionsController(private val visitSchedulerService: VisitSc
     prisonerId: String,
     @RequestParam(value = "sessionRestriction", required = false)
     @Parameter(description = "Filter sessions by session restriction - OPEN or CLOSED, if prisoner has CLOSED it will use that", example = "CLOSED", required = false)
-    sessionRestriction: SessionRestriction = SessionRestriction.OPEN,
+    sessionRestriction: SessionRestriction?=null,
```

Comment:
why are we doing this?

Reply:
it defaults later in the code

Reply:
val sessionRestriction = if (hasClosedRestriction(restrictions)) CLOSED else requestedSessionRestriction ?: OPEN